bin/preview-docs

```diff
@@ -48,78 +47,6 @@ while test $# -gt 0; do
   esac
 done
 
-# Create the output directory for the symbol graphs if needed.
```

Comment:
I guess this means contributors will need Swift 5.6 to be able to build and preview documentation for Swift-DocC? Should we keep the previous way of building docs around for contributors that are still developing on 5.5 for a small period of time?

Reply:
I agree with this.

Reply:
I agree it makes sense to move incrementally with this. Given that, I also updated our README to suggest people use Swift-DocC-Render-Artifact instead of building Swift-DocC-Render locally.

[efffe2a](https://github.com/apple/swift-docc/pull/106/commits/efffe2a0a268dcab69c6f15ea8baaf7ae1449417)